src/sentry/api/endpoints/organization_release_details.py

```diff
@@ -129,7 +129,7 @@ def put(self, request, organization, version):
         if not self.has_release_permission(request, organization, release):
             raise PermissionDenied
 
-        serializer = ReleaseSerializer(data=request.DATA, partial=True)
+        serializer = ReleaseSerializer(data=request.DATA)
```

Comment:
i don't think changing this will be a problem since all of the fields are `required=False` on the `ReleaseSerializer`